### PR TITLE
feat: add Meter testnet

### PR DESCRIPTION
**Pull request by JaunePomme**
### Description

Add Meter testnet
Found information on the Meter explorer and its doc.
### Backward compatibility

No
### Testing

Yes

### CODE REVIEW

File: chains/metertestnet/logo.svg [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+46 96 46 2 0 1 -14 -1 -32 l-5 -32 49 29 c27 16 64 38 83 49 37 22 35 26 16
+-41 -3 -13 -4 -23 0 -23 8 0 245 131 245 135 0 2 -39 -9 -87 -26 -48 -16 -89
+-29 -91 -29 -3 0 -2 18 2 40 3 22 3 40 0 40 -4 0 -40 -18 -81 -41 l-74 -41 7
+41 c4 23 3 41 -2 40 -5 0 -49 -34 -98 -75z"/>
+<path d="M568 337 c-9 -6 -21 -35 -28 -64 -45 -198 -294 -243 -412 -75 -12 18
+-22 22 -37 17 -12 -3 -21 -10 -21 -15 0 -4 20 -30 45 -58 60 -66 119 -94 205
+-94 83 0 143 27 199 87 45 49 65 91 73 151 7 51 -2 69 -24 51z"/>
+</g>
+</svg>
+
```
Comment (paulbalaji) [inline chains/metertestnet/logo.svg:23]: only need svg tags + minified on https://jakearchibald.github.io/svgomg/
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" version="1.0" viewBox="0 0 64 64"><path d="M27.5 5c-6.1 1-9.1 2.6-13.9 7-6.5 6-9.4 13.4-8.7 22.4.2 3.7 3.7-1 5-6.8C14.5 7.2 39 2.9 51.5 20.3c1.4 2 2.6 2.6 3.8 2.1 2.7-1 2.1-2.8-2.8-8.2-6.8-7.6-15.4-10.7-25-9.2z"/><path d="M17.6 29.6c-4.8 4.1-10.2 8.6-12 10-2.3 1.9-.8 1.4 6.1-2 5.1-2.5 9.4-4.6 9.6-4.6.2 0 .1 1.4-.1 3.2l-.5 3.2 4.9-2.9c2.7-1.6 6.4-3.8 8.3-4.9 3.7-2.2 3.5-2.6 1.6 4.1-.3 1.3-.4 2.3 0 2.3.8 0 24.5-13.1 24.5-13.5 0-.2-3.9.9-8.7 2.6-4.8 1.6-8.9 2.9-9.1 2.9-.3 0-.2-1.8.2-4 .3-2.2.3-4 0-4-.4 0-4 1.8-8.1 4.1l-7.4 4.1.7-4.1c.4-2.3.3-4.1-.2-4-.5 0-4.9 3.4-9.8 7.5z"/><path d="M56.8 30.3c-.9.6-2.1 3.5-2.8 6.4-4.5 19.8-29.4 24.3-41.2 7.5-1.2-1.8-2.2-2.2-3.7-1.7-1.2.3-2.1 1-2.1 1.5 0 .4 2 3 4.5 5.8 6 6.6 11.9 9.4 20.5 9.4 8.3 0 14.3-2.7 19.9-8.7 4.5-4.9 6.5-9.1 7.3-15.1.7-5.1-.2-6.9-2.4-5.1z"/></svg>
```

File: chains/metertestnet/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://testnet.meter.io
+    family: EVM
+    name: Meter Testnet Explorer
+    url: https://scan.meter.io/
```
Comment (paulbalaji) [inline chains/metertestnet/metadata.yaml:6]:
```suggestion
    url: https://scan-warringstakes.meter.io
```

File: chains/metertestnet/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://testnet.meter.io
+    family: EVM
```
Comment (paulbalaji) [inline chains/metertestnet/metadata.yaml:4]:
```suggestion
    family: other
```

File: chains/metertestnet/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://testnet.meter.io
```
Comment (paulbalaji) [inline chains/metertestnet/metadata.yaml:3]: is there an api for verifying contracts?

Comment (JaunePomme) [reply chains/metertestnet/metadata.yaml:3]: From what I've found, unfortunately, Meter (mainnet and testnet) uses Sourcify for contract verification and there is no clear api endpoint for that; It looks like it's done through web interfaces (Sourcify portal/Meter explorer..)

File: chains/metertestnet/metadata.yaml (hunk 2 above)
Comment (JaunePomme) [reply chains/metertestnet/metadata.yaml:6]: Yes, thanks

File: chains/metertestnet/logo.svg (hunk 1 above)
Comment (JaunePomme) [reply chains/metertestnet/logo.svg:23]: Thanks, I had troubles with this svg file.

**Review by jmrossy — APPROVED** [review]

File: chains/metertestnet/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    family: other
+    name: Meter Testnet Explorer
+    url: https://scan-warringstakes.meter.io
+chainId: 83
+deployers:
```
Comment (paulbalaji) [inline chains/metertestnet/metadata.yaml:8]:
```suggestion
deployer:
```

**Comment by jmrossy** [comment]
@JaunePomme the prettier test is failing. To fix it, run `yarn prettier` and that will format your files correctly

File: chains/metertestnet/metadata.yaml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+domainId: 83
+isTestnet: true
+name: metertestnet
+nativeToken:
+  decimals: 18
+  name: Ether
+  symbol: ETH
+protocol: ethereum
+rpcUrls:
+  - http: https://rpctest.meter.io
```
Comment (paulbalaji) [inline chains/metertestnet/metadata.yaml:21]:
```suggestion
# yaml-language-server: $schema=../schema.json
blockExplorers:
  - apiUrl: https://testnet.meter.io
    family: other
    name: Meter Testnet Explorer
    url: https://scan-warringstakes.meter.io
chainId: 83
deployer:
  name: JaunePomme
  url: https://github.com/JaunePomme
displayName: Meter testnet
domainId: 83
isTestnet: true
name: metertestnet
nativeToken:
  decimals: 18
  name: Ether
  symbol: ETH
protocol: ethereum
rpcUrls:
  - http: https://rpctest.meter.io

```

Comment (paulbalaji) [inline chains/metertestnet/metadata.yaml:21]:
```suggestion
# yaml-language-server: $schema=../schema.json
blockExplorers:
  - apiUrl: https://testnet.meter.io
    family: other
    name: Meter Testnet Explorer
    url: https://scan-warringstakes.meter.io
chainId: 83
deployer:
  name: JaunePomme
  url: https://github.com/JaunePomme
displayName: Meter testnet
domainId: 83
isTestnet: true
name: metertestnet
nativeToken:
  decimals: 18
  name: Ether
  symbol: ETH
protocol: ethereum
rpcUrls:
  - http: https://rpctest.meter.io

```

**Comment by JaunePomme** [comment]
Yes thx, fixed my prettier (again)

File: chains/metertestnet/metadata.yaml (hunk 3 above)
Comment (paulbalaji) [inline chains/metertestnet/metadata.yaml:4]:
```suggestion
    family: other
```

**Pull request merged**